meinberlin/apps/cms/templates/meinberlin_cms/blocks/map_teaser_block.html

```diff
@@ -1,9 +1,6 @@
 {% load wagtailimages_tags wagtailcore_tags block_tags react_map_teaser static %}
 
-{% block extra_js %}
-    <script type="text/javascript" src="{% static 'react_map_teaser.js' %}"></script>
-    {{ block.super }}
-{% endblock %}
+<script type="text/javascript" src="{% static 'react_map_teaser.js' %}"></script>
```

Comment:
the extra js block not needed here?

Reply:
this template doesn't extent any other template, so there is no super block, and no other templates extends this, so the block is redundant afaic

Reply:
I'm not sure if having a script tag somewhere in the body is good practice though, what do you think @philli-m ?

Reply:
I don't understand where are these blocks like the map_teaser_block here included?

Reply:
@goapunk I guess it would be best to add it in the extra js tag for the home_page, I assume that would work? although django templates are bit weird about whats loaded when?!
@m4ra the mB homepage is created via the editors in wagtail, that is where the map_teaser_block is added

Reply:
the map_teaser_block is a wagtail block (https://github.com/liqd/a4-meinberlin/blob/dev/meinberlin/apps/cms/blocks.py#L117)  which can be added via wagtail. I think it will then be used here https://github.com/liqd/a4-meinberlin/blob/dev/meinberlin/apps/cms/templates/meinberlin_cms/home_page.html#L69. Does that help?

Reply:
@goapunk yes thanks. the extra js block then can definitely go in the bottom of the home_page.